packages/flutter_test/lib/src/window.dart

```diff
@@ -944,3 +982,252 @@ class TestPlatformDispatcher implements ui.PlatformDispatcher {
     return null;
   }
 }
+
+/// [ui.FlutterView] that wraps another [ui.FlutterView] and allows faking of
+/// some properties for testing purposes.
+///
+/// See also:
+///
+///   * [TestPlatformDispatcher], which allows for faking of platform specific
+///     functionality.
+class TestFlutterView implements ui.FlutterView {
```

Comment:
I actually can't unless I make it a part file. Part of supporting the `TestWindow` API for backwards compatibility requires a private API on `TestFlutterView`.

Reply:
Can you rename this file to view.dart then (which should be the new permanent home after all the deprecated stuff is gone)? I am really trying to purge anything that uses the term window to mean "view" from our code base (or at least deprecate it if it is load-baring) to avoid any unnecessary confusion when we introduce actual windows.

Reply:
Good idea, but I'm going to do that in a different PR as internal files _do_ reference this file by name directly. I'll add it to my (growing) list of issues to file.